### PR TITLE
Download zip from a url

### PR DESCRIPTION
**Pull request by albertolerda**

### CODE REVIEW

File: packages/files/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { NextFunction, Request, Response } from "express";
+import fs from 'fs'
+import https from "https";
+import {
+  DOWNLOAD,
+} from "./actions";
+import yauzl from 'yauzl';
+import extract from 'extract-zip';
+
+const TEMP_FILE = '/tmp/temp-zip-file-downloaded-from-restroom';
```
Comment (puria) [inline packages/files/src/index.ts:14]: Please add a TODO to fix this (no hardcoded) with `fs.mkdtemp` or `os.tempdir` so it's compatible with all OSes

File: packages/files/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  let content: ObjectLiteral = {};
+
+  rr.onBefore(
+    async (params: {
+      zencode: Zencode;
+      keys: ObjectLiteral;
+      data: ObjectLiteral;
+    }) => {
+    }
+  );
```
Comment (puria) [inline packages/files/src/index.ts:27]: We can remove this... is useless right?

File: packages/files/src/index.ts (hunk 1 above)
Comment (puria) [reply packages/files/src/index.ts:14]: This is a major issue if you run like a restroom with a application server like `pm2` that run more concurrent processes for one app, and if you have many processes writing in the same place it'll be sad 🤣

**Review by puria — CHANGES_REQUESTED** [review]

File: packages/files/src/index.ts (hunk 2 above)
Comment (andrea-dintino) [reply packages/files/src/index.ts:27]: Yes it should be, this mw should operate only in on.success

**Review by puria — APPROVED** [review]

**Pull request merged**